content/Circular_Intersection.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Circular Intersections
+draft: true
+tags:
+  - math
+---
+# Circular Intersections
+## How I started this problem
+One day I was writing my school HW where I had to fill out a venn diagram contrasting to people. Because the middle section soon ran out of space, I thought: *How can I make all three sections the same?*
+## Problem
+When taking two congruent circles, when are the area's of the intersection of the two circles equal to the un intersected section of one circle?
```

Comment:
here we need some transition sentences to transit from prev section to this section; or we could add transition sentences in the end of prev section which would lead to this Problem section: e.g.,  ``I thought: *How can I make all three sections the same?* This can easily defined in mathematical form as described in next section. ''